tests/test_migrations.py

```diff
@@ -54,15 +65,22 @@ def test_migrations_and_tasks(tmp_path: Path) -> None:
     assert answers == {"_commit": "v1.0.0", "_src_path": str(src)}
     # Save changes in downstream repo
     with local.cwd(dst):
+        git("init")
         git("add", ".")
         git("config", "user.name", "Copier Test")
         git("config", "user.email", "test@copier")
         git("commit", "-m1")
     # Update it to v2
-    run_update(dst_path=dst, defaults=True, overwrite=True, unsafe=True)
+    run_update(
+        dst_path=dst, defaults=True, overwrite=True, unsafe=True, skip_tasks=skip_tasks
+    )
     # Check update was OK
-    assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n" * 2
-    assert not (dst / "delete-in-tasks.txt").exists()
+    if not skip_tasks:
+        assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n" * 2
+        assert not (dst / "delete-in-tasks.txt").exists()
+    else:
+        assert not (dst / "created-with-tasks.txt").exists()
+        assert (dst / "delete-in-tasks.txt").exists()
```

Comment:
Same suggestion here:

```suggestion
    if skip_tasks:
        assert not (dst / "created-with-tasks.txt").exists()
        assert (dst / "delete-in-tasks.txt").exists()
    else:
        assert (dst / "created-with-tasks.txt").read_text() == "task 1\ntask 2\n" * 2
        assert not (dst / "delete-in-tasks.txt").exists()
```